packages/siri-sx-generator/package.json

```diff
@@ -8,7 +8,9 @@
     },
     "devDependencies": {
         "@types/aws-lambda": "^8.10.110",
+        "@types/jstoxml": "^2.0.2",
         "@types/node": "^18.13.0",
+        "jstoxml": "^3.2.6",
```

Comment:
This would be better off as a dependency rather than a devDependency

Reply:
done